### ISSUE TITLE
[BUG] - outward DE wont start 

### ISSUE THREAD

**Issue by UrsoDeTerno** [ebkr/r2modmanPlus#977]
since the latest update i cant start outward DE moded or even valina using the mod manager, when i try to  start i get this message
![Sem título](https://user-images.githubusercontent.com/126001382/220402476-4fb61160-93c1-4bd2-bf10-4c329845d332.png)

"outward is trying to iniciate with the optinals parameters bellow"
if i click "ok" steam starts normal vanila outward and not moded outward DE

**Comment by ZeroZone8** [comment]
I also have this issue, since the last update, starting a game from the mod manager goes to regular Outward and not DE Outward. 

While it doesn't fix the issue with the mod manager, a work around I just found was to use the "Launching the game from outside the mod manager" argument provided in the "Help -> General" section. 
1. Copy the argument 
2. In Steam right click Outward and go to the Properties
3. Under the "Advanced users may choose to enter modifications to their launch options." Paste the copied argument.
4. Launch Outward from Steam.

At least for me, this has allowed me to continue my modded playthrough of Outward DE.

**Comment by UrsoDeTerno** [comment]
it lauches, but without the mods

**Comment by fred777** [comment]
does not work for me either. via mod manager it will run regular Outward.

And adding launch options to Steam as suggested by ZeroZone8 will run vanilla Def Edition without mods in my case.

**Comment by fred777** [comment]
Update: the contents of AppData\Roaming\r2modmanPlus-local\OutwardDe\profiles\Default\BepInEx\LogOutput.log are truncated - maybe due to a crash in the patcher?
```
[Message:   BepInEx] BepInEx 5.4.19.0 - Outward Definitive Edition (19.02.2023 13:19:57)
[Info   :   BepInEx] Running under Unity v2020.3.26.7510196
[Info   :   BepInEx] CLR runtime version: 4.0.30319.42000
[Info   :   BepInEx] Supports SRE: True
[Info   :   BepInEx] System platform: Bits64, Windows
[Message:   BepInEx] Preloader started
[Info   :   BepInEx] Loaded 1 patcher method from [BepInEx.Preloader 5.4.19.0]
[Info   :   BepInEx] 1 patcher plugin loaded
[Info   :   BepInEx] Patching [UnityEngine.CoreModule] with [BepInEx.Chainloader]
[Message:   BepInEx] Preloader finished
[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 4 plugins to load
[Info   :   BepInEx] Loading [Combat Tweaks 3.2.0]
[Info   :   BepInEx] Loading [SideLoader 3.8.4]
[Message:SideLoader] Version 3.8.4 starting...
[Info   :   BepInEx] Loading [Repair inventory after rest 1.0.0]
[Info   :   BepInEx] Loading [ActionUI 1.1.5]
[Info   :  ActionUI] Patching...
[Info   : 
```

**Comment by fred777** [comment]
Not sure if this related: In the code, Def Edition is referenced with Steam ID 794260 - which is actually the same as for Non-Def Edition: https://github.com/ebkr/r2modmanPlus/blob/master/src/model/game/GameManager.ts#L79

But the correct ID for Def Edition seems to be 1758860, see https://www.youtube.com/watch?v=Tw-1UL1kW0A for reference.

**Comment by ebkr** [comment]
This should now be fixed in the latest release (3.1.38)

**Comment by ebkr** [comment]
Closing as no response for a long time
